### PR TITLE
Modified documents related with mknb

### PR DESCRIPTION
**Pull request by xuweibj**
1. run mknb if changing the ip of MN.
http://10.3.5.4/install/build/html/advanced/cluster_maintenance/mgmt_node/changing_hostname_ip.html
2. verify genesis packages before provision for mixed_cluster.
http://10.3.5.4/install/build/html/advanced/mixed_cluster/power/diskful.html

### CODE REVIEW

File: docs/source/advanced/mixed_cluster/power/diskful.rst [hunk 1]
```diff
@@ -44,6 +44,15 @@ Create a node definition for the x86_64 compute node, here is a sample: ::
         serialport=0
         serialspeed=115200
 
+Verify the genesis packages:
+
+* **[RHEL/SLES]**: ``rpm -qa | grep -i genesis``
+
+* **[Ubuntu]**: ``dpkg -l | grep -i genesis``
+
+If missing, install them from the ``xcat-deps`` package and run ``mknb x86_64`` to create the genesis network boot root image.
```
Comment (daniceexi) [inline docs/source/advanced/mixed_cluster/power/diskful.rst]: Suggest to 
~~~~
If missing, install the packages ``xCAT-genesis-base'' and ``xCAT-genesis-scripts`` from ``xcat-deps`` repository and run ``mknb <arch>`` to ...
~~~~

File: docs/source/advanced/cluster_maintenance/mgmt_node/changing_hostname_ip.rst [hunk 2]
```diff
@@ -270,3 +270,8 @@ If it exists, then use the return name and do the following:
 
       makeconservercf
 
+Update the genesis packages
+---------------------------
+
+Run ``mknb <arch>`` after changing the ip of MN.
```
Comment (whowutwut) [inline docs/source/advanced/cluster_maintenance/mgmt_node/changing_hostname_ip.rst]: Is this necessary? 

If changing the IP, would we also have to change the networks table entry, what happens if the IP is in completely new subnets.  Should we also remove other entries in the networks table to clean up?

File: docs/source/advanced/mixed_cluster/power/diskful.rst (hunk 1 above)
Comment (whowutwut) [inline docs/source/advanced/mixed_cluster/power/diskful.rst]: :+1:

File: docs/source/advanced/cluster_maintenance/mgmt_node/changing_hostname_ip.rst (hunk 2 above)
Comment (daniceexi) [inline docs/source/advanced/cluster_maintenance/mgmt_node/changing_hostname_ip.rst]: I think run of the ``mknb`` command is just in the case the network also is changed so the mknb command will update the genesis boot cfg file for discovery.

**Comment by daniceexi** [comment]
We can merge the change first, we could refine it if have better,

**Pull request merged**